### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by agrawalshanu708**
create Authentication
user can signup and login

### CODE REVIEW

**Review by tanishk15sharma — APPROVED** [review]

**Review by MonikaJhingan — APPROVED** [review]
The code structure looks clean. Needs to be merged.

File: src/reduxManagement/slice/auth/authService.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+export const signupHandler = createAsyncThunk(
+  "auth/signupHandler",
+  async ({ firstName, lastName, username, password }, thunkAPI) => {
+    console.log(firstName, lastName, username, password);
```
Comment (rahulyadav139) [inline src/reduxManagement/slice/auth/authService.js:6]: remove comments.

File: src/screens/Auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    password: "",
+  });
+   const dispatch = useDispatch()
+  const clickHandler = () => {
+      dispatch(signupHandler({
+        firstName: signupCredential.firstName,
+        lastName: signupCredential.lastName,
+        username: signupCredential.username,
+        password: signupCredential.password
+      }))
```
Comment (rahulyadav139) [inline src/screens/Auth/Signup.jsx:29]: Suggestion: you can use object destructuring here.

**Review by rahulyadav139 — APPROVED** [review]

**Review by shashikantmaurya0007 — COMMENTED** [review]
Code looks good to be merged

**Pull request merged**